common/app/views/fragments/immersiveGalleryHeadTonal.scala.html

```diff
@@ -0,0 +1,90 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive content__head--desktop is-fixed-height tonal__head tonal__head--@toneClass(item)">
```

Comment:
A new module is an interesting idea. Would this be just to share things between projects like article, applications, and facia and not compile to other applications?

Does it makes more sense in this instance to try to combine these to files with the necessary if statements in common, or have too similar files one in article, and one in applications.